packages/authentication/src/types.ts

```diff
@@ -133,3 +133,16 @@ export const asAuthStrategy: BindingTemplate = binding => {
       AuthenticationBindings.AUTHENTICATION_STRATEGY_EXTENSION_POINT_NAME,
   });
 };
+
+/**
+ * Get the authentication metadata object for the specified strategy.
+ *
+ * @param metadata - Array of authentication metadata objects
+ * @param strategyName - Name of the authentication strategy
+ */
+export function getAuthenticationMetadataForStrategy(
```

Comment:
@jannyHou Its just a helper function to be used by the consumer of the library. One drawback now is that the injected metadata will be an array which means it is required to get the specific metadata for the strategy and this function can be used to do this. I just put it into `types.ts` because there were most other functions as well but its probably a good idea to create a file called `utils.ts` or `helpers.ts` were those functions should be placed. Same goes for the `registerAuthenticationStrategy` function.

Reply:
> since there are more functions that should be moved to different files it might make sense to do it in another refactoring PR

@nflaig Thank you for the explanation, the plan sounds good to me.